### PR TITLE
[RemoteEvent][Webhook] Add Sendinblue support

### PR DESCRIPTION
**Pull request by blaugueux**
| Q             | A
| ------------- | ---
| Branch       | 6.4
| Bug fix      | no
| New feature  | yes
| Deprecations | no
| Tickets       | 
| License       | MIT
| Doc PR        |

### CODE REVIEW

File: src/Symfony/Component/Mailer/Bridge/Sendinblue/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+6.4.0
+-----
```
Comment (OskarStark) [inline src/Symfony/Component/Mailer/Bridge/Sendinblue/CHANGELOG.md]:
```suggestion
6.4
---
```

File: src/Symfony/Component/Mailer/Bridge/Sendinblue/Webhook/SendinblueRequestParser.php [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        }
+
+        try {
+            return $this->converter->convert($content);
+        } catch (ParseException $e) {
+            throw new RejectWebhookException(406, $e->getMessage(), $e);
+        }
+    }
+
+
```
Comment (OskarStark) [inline src/Symfony/Component/Mailer/Bridge/Sendinblue/Webhook/SendinblueRequestParser.php]:
```suggestion
```

**Comment by blaugueux** [comment]
@OskarStark Can you please tell me if I need to work on something on this PR?

**Comment by OskarStark** [comment]
Looks good from my side, WDYT @fabpot ?

**Comment by stof** [comment]
Given that Sendinblue has rebranded as Brevo and there is an in-progress PR to create a Brevo bridge using the new domain, I would vote -1 for this PR in its current state. The webhook and remote event integrations should be added to the new bridge, not to the old one.

**Comment by fabpot** [comment]
@blaugueux Can you work on adding this to the new bridges?

**Comment by blaugueux** [comment]
@fabpot of course. Just give me few days!

**Comment by blaugueux** [comment]
@fabpot https://github.com/symfony/symfony/pull/51264 integration into the new Brevo bridge.

**Comment by fabpot** [comment]
Thank you, let's close here then.

**Pull request closed without merging**